.github/workflows/ci.yml

```diff
@@ -101,6 +102,10 @@ jobs:
       - name: Run Flake8
         run: |
           flake8 --color never --count --config .flake8 .
+
+      - name: Run MyPy
+        run: |
+           mypy --no-warn-unused-ignores
```

Comment:
I think this is better handles in the `mypy.ini` as `warn_unused_ignores = False`. Makes it easier for people to run mypy locally.

Reply:
Good point - updated.

Yes if it gets too annoying I will disable it.